### ISSUE TITLE
Clarify on location of .bin file

### ISSUE THREAD

**Issue by Iceman253** [spiritualized1997/openFPGA-GB-GBC#53]
Hi, just wondering where I can find the gbc_bios file. Do I just create an empty file?
And do I need a different binary file for the gameboy rom.

**Comment by MaciekBaron** [comment]
No, you need a GBC bios file, not an empty file. The official BIOS is copyrighted, so obviously nobody will link to it. Try "acquiring it legally" or googling it.
